src/styles/todo.scss

```diff
@@ -28,11 +28,20 @@
     opacity: 0;
   }
 
+  &-loader {
+    opacity: 65%;
```

Comment:
Opacity value should be between 0 and 1. Change the value to a valid range, e.g., 0.65.